subprojects/packagefiles/cmark-gfm/meson.build

```diff
@@ -0,0 +1,121 @@
+project(
+    'cmark-gfm',
+    'c',
+    version: '0.29.0.gfm.6',
+    default_options: ['warning_level=3'],
+)
+
+cc = meson.get_compiler('c')
+has_stdbool_header = cc.has_header('stdbool.h')
+has_builtin_expect_code = 'int main() { __builtin_expect(0,0); return 0; }'
+has_builtin_expect = cc.links(
+    has_builtin_expect_code,
+    name: 'has __builtin_expect',
+)
+has_attribute_code = '''
+  int f(void) __attribute__ (());
+  int main() { return 0; }
+'''
+has_attribute = cc.links(has_attribute_code, name: 'has __attribute__')
```

Comment:
I suspect it's probably enough to check if these `cc.compiles()`, without also running the linker.

Reply:
Agreed, fixed.